web/src/main/java/org/cbioportal/web/parameter/CNAFilter.java

```diff
@@ -0,0 +1,25 @@
+package org.cbioportal.web.parameter;
+
+import org.cbioportal.model.CNA;
+
+public class CNAFilter {
```

Comment:
Studies can have multiple CNA profiles (ex: gbm_tcga_pub study has 2 CNA profiles). `profileType` needs to be included. It looks much similar to `GenomicDataFilter`, may be reuse it